src/main/java/com/codejoust/main/Main.java

```diff
@@ -30,6 +30,7 @@ public static void main(String[] args) {
 			log.error("Could not find the firebase keyfile in an environment variable.");
 			log.error("Please set this value before launching the program.");
 			System.exit(1);
+			return;
```

Comment:
Adding this `return` statement, which may be superfluous but removed the `Potential null pointer access: The variable keyfile may be null at this location` warning from `java.compile.nullAnalysis.mode`.